pyproject.toml

```diff
@@ -46,10 +46,16 @@ dependencies = [
     'numpy',
     'pyyaml',
     'torch',
+    'matplotlib',
+    'pytorch_seed',
 ]
 
 [project.optional-dependencies]
-test = ["pytest"]
+test = [
+    "pytest",
+    "pybullet",
+    "jax",
```

Comment:
do we actually need this? I don't see it being used

Reply:
This has been removed a couple commits back (only needed for mujoco menagerie)